--- support/misc/toolchainfile.cmake.in.orig	2018-04-20 17:06:35.998586370 +0200
+++ support/misc/toolchainfile.cmake.in	2018-04-20 17:07:12.271698565 +0200
@@ -58,8 +58,8 @@
 set(ENV{PKG_CONFIG_SYSROOT_DIR} "${RELOCATED_HOST_DIR}/arm-buildroot-linux-uclibcgnueabihf/sysroot")
 
 # This toolchain file can be used both inside and outside Buildroot.
-set(CMAKE_C_COMPILER "${RELOCATED_HOST_DIR}/bin/arm-buildroot-linux-uclibcgnueabihf-gcc")
-set(CMAKE_CXX_COMPILER "${RELOCATED_HOST_DIR}/bin/arm-buildroot-linux-uclibcgnueabihf-g++")
+set(CMAKE_C_COMPILER "${RELOCATED_HOST_DIR}/bin/arm-buildroot-linux-uclibcgnueabihf-gcc" CACHE STRING "C")
+set(CMAKE_CXX_COMPILER "${RELOCATED_HOST_DIR}/bin/arm-buildroot-linux-uclibcgnueabihf-g++" CACHE STRING "CXX")
 if(0)
   set(CMAKE_Fortran_FLAGS_DEBUG "" CACHE STRING "Debug Fortran FLAGS")
   set(CMAKE_Fortran_FLAGS_RELEASE " -DNDEBUG" CACHE STRING "Release Fortran FLAGS")
